grudge/discretization.py

```diff
@@ -156,7 +160,7 @@ def _set_up_distributed_communication(self, mpi_communicator, array_context):
         return boundary_connections
 
     def get_distributed_boundary_swap_connection(self, dd):
-        if dd.quadrature_tag != QTAG_NONE:
+        if dd.discretization_tag not in (DISCR_TAG_BASE, DISCR_TAG_MODAL):
             # FIXME
             raise NotImplementedError("Distributed communication with quadrature")
```

Comment:
If you ask `connection_from_dds` for a connection modal volume -> nodal boundary, does that work? Also, can you ask for a "face->volume mass matrix" onto the modal discretization? (If the answer to either of these is no, then this shouldn't suggest that modal is allowed here.)

Reply:
Good point, I removed it in this commit: https://github.com/inducer/grudge/pull/83/commits/f5f016ef9b81fdbc951ec002130db393d67d0894 (And I don't think you can do either of those things currently, so I'll err on the safe side for now)